app/domain/content/importers/content_details.rb

```diff
@@ -0,0 +1,17 @@
+module Content
+  class Importers::ContentDetails
+    attr_reader :items_service, :content_id, :base_path
+
+    def initialize(content_id, base_path)
+      @content_id = content_id
+      @base_path = base_path
+      @items_service = ItemsService.new
+    end
+
+    def run
+      response = items_service.fetch_raw_json_from_content_store(base_path)
```

Comment:
The method name should not expose where the date is coming from:
What about renaming it to: ```items_service.get_raw_json(base_path)```